packages/block-library/src/post-navigation-link/edit.js

```diff
@@ -94,70 +101,98 @@ export default function PostNavigationLinkEdit( {
 	return (
 		<>
 			<InspectorControls>
-				<PanelBody>
-					<ToggleControl
-						__nextHasNoMarginBottom
+				<ToolsPanel
+					label={ __( 'Settings' ) }
+					dropdownMenuProps={ dropdownMenuProps }
+				>
+					<ToolsPanelItem
+						hasValue={ () => !! showTitle }
 						label={ __( 'Display the title as a link' ) }
-						help={ __(
-							'If you have entered a custom label, it will be prepended before the title.'
-						) }
-						checked={ !! showTitle }
-						onChange={ () =>
-							setAttributes( {
-								showTitle: ! showTitle,
-							} )
+						onDeselect={ () =>
+							setAttributes( { showTitle: false } )
 						}
-					/>
-					{ showTitle && (
+						isShownByDefault
+					>
 						<ToggleControl
 							__nextHasNoMarginBottom
-							label={ __(
-								'Include the label as part of the link'
+							label={ __( 'Display the title as a link' ) }
+							help={ __(
+								'If you have entered a custom label, it will be prepended before the title.'
 							) }
-							checked={ !! linkLabel }
+							checked={ !! showTitle }
 							onChange={ () =>
 								setAttributes( {
-									linkLabel: ! linkLabel,
+									showTitle: ! showTitle,
 								} )
 							}
 						/>
+					</ToolsPanelItem>
+					{ showTitle && (
+						<ToolsPanelItem
+							hasValue={ () => !! linkLabel }
+							label={ __( 'Include the label as part of the link' ) }
+							onDeselect={ () => setAttributes( { linkLabel: false } ) }
+						>
+							
+								<ToggleControl
+									__nextHasNoMarginBottom
+									label={ __(
+										'Include the label as part of the link'
+									) }
+									checked={ !! linkLabel }
+									onChange={ () =>
+										setAttributes( {
+											linkLabel: ! linkLabel,
+										} )
+									}
+									isShownByDefault
+								/>
+							
+						</ToolsPanelItem>
 					) }
-					<ToggleGroupControl
-						__next40pxDefaultSize
-						__nextHasNoMarginBottom
+					<ToolsPanelItem
+						hasValue={ () => !! arrow }
```

Comment:
```suggestion
						hasValue={ () => arrow !== 'none' }
```

If the value is the default value `none`,  we must return `false`.